test/sequential/test-timers-block-eventloop.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+const http = require('http');
+const assert = require('assert');
+
+const t1 = setInterval(() => {
+  common.busyLoop(12);
+}, 10);
+
+const t2 = setInterval(() => {
+  common.busyLoop(15);
+}, 10);
+
+const t3 = setTimeout(() => {
+  throw new Error('eventloop blocked!');
+}, 200).unref();
+
+const server = http.createServer((req, res) => {
+  res.end('ok');
+  server.close();
+}).listen(0, common.mustCall(() => {
+  http.request({
+    port: server.address().port,
+    encoding: 'utf8'
+  }, common.mustCall((res) => {
+    let data = '';
+    res.on('data', (chunk) => data += chunk);
+    res.on('end', common.mustCall(() => {
+      assert.strictEqual(data, 'ok');
+      clearInterval(t1);
+      clearInterval(t2);
+      clearInterval(t3);
+    }));
+  })).end();
+}));
```

Comment:
I am not sure why the server is included in the test. And if I am correct this could actually obfuscate what we really want to test if the server responds in below than 10 milliseconds.

Please remove it therefore and find a simpler test case.

Reply:
simpler test case updated.